Example/SnapshotTests/TextInputsTests.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  Copyright 2021 Square Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import AccessibilitySnapshot
+import FBSnapshotTestCase
+
+final class TextInputsTests: SnapshotTestCase {
```

Comment:
Instead of having a separate test for each type of input, it might be beneficial to snapshot the `TextInputsViewController` as a whole. That makes it super easy to compare the reference images to the demo app to confirm they match VO behavior, e.g. when a new iOS version comes out.